dart/lib/src/sentry_options.dart

```diff
@@ -136,6 +136,15 @@ class SentryOptions {
   /// example : ['sentry'] will include exception from 'package:sentry/sentry.dart'
   List<String> get inAppIncludes => List.unmodifiable(_inAppIncludes);
 
+  /// Configures wether stack trace frames are considered in app frames as default.
+  /// You can use this to essentially make [inAppIncludes] or [inAppExcludes]
+  /// an allow or deny list.
+  /// This value is only used if Sentry can not find the origin of the frame.
+  ///
+  /// E.g. if [isStackFrameInAppDefault] is true you only need to maintain
+  /// [inAppExcludes].
+  bool isStackFrameInAppDefault = true;
```

Comment:
```suggestion
  bool inAppFramesByDefault = true;
```

this sounds better to me tbh